include/deal.II/base/complex_overloads.h

```diff
@@ -63,6 +78,22 @@ operator*(const std::complex<T> &left, const U &right)
   return static_cast<result_type>(left) * static_cast<result_type>(right);
 }
 
+/**
+ * Provide an <tt>operator/</tt> for a scalar division of a complex
+ * floating point type with a different real floating point type. Annoyingly,
+ * the standard library does not provide such an operator...
+ *
+ * @relatesalso EnableIfScalar
+ * @relatesalso ProductType
+ */
+template <typename T, typename U>
+typename ProductType<std::complex<T>,
+                     typename EnableIfScalar<U>::type>::type inline
```

Comment:
same here

Reply:
is `EnableIfScalar` false for complex types? I tend to think of complex types as scalars as well...

Reply:
It actually is: https://github.com/dealii/dealii/blob/master/include/deal.II/base/template_constraints.h#L567-L571, but 
```
operator/(const std::complex<T>&, const std::complex<U>&)
```
is a better match: https://godbolt.org/z/W4ikrT

Reply:
I see, thank you!